Lilianada/Fitcreatives
# Update data in JSON file

### This PR introduces gym-related data into the data.json file. 

### Testing Instructions:
- Pull this branch.
- Open the data.json file to review the added data.
- Review and remove comments.
- Run the app and confirm that the gym data is displayed correctly in related features.
